src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -43,15 +43,50 @@ public void testVCFModeIsConsistentWithPastResults() throws Exception {
         IntegrationTestSpec.assertEqualTextFiles(output, expected);
     }
 
+
     /*
-     * Test that in VCF mode we're >= 99% concordant with GATK3.4 results
+     * Test that in VCF mode we're consistent with past GATK4 results
      */
     @Test
-    public void testVCFModeIsConcordantWithGATK3Results() throws Exception {
+    public void testVCFModeIsConsistentWithPastResults_AlleleSpecificAnnotations() throws Exception {
+        Utils.resetRandomGenerator();
+
+        //NOTE: AlleleSpecific support in the VCF mode is implemented but bous for now.
```

Comment:
`bous` -> `bogus`. Also, if the AS output in VCF mode is bogus, we should probably be throwing a `UserException` for now in `HaplotypeCaller` when the user requests that combination, and open a ticket to address.